components/measurement/nettests/HTTPHeaderFieldManipulation.js

```diff
@@ -32,7 +32,7 @@ export const HttpHeaderFieldManipulationDetails = ({ measurement, render }) => {
       details: (
         <div>
           {/*<Text>isAnomaly: {isAnomaly.toString()}</Text>
-          <Text>isFailed: {isFailed.toString()}</Text>
+            <Text>isFailed: {isFailed.toString()}</Text>
```

Comment:
Same as above. I'd like to encourage you to not keep commented out code. Commented out code rots twice as fast as comments describing implementations, so I'd be mostly pleased if we can avoid it.